apps/coscrad-frontend/src/components/resources/spatial-features/constants/index.ts

```diff
@@ -1,5 +1,16 @@
 import { Position2D } from '../types';
 
+/**
+ * NOTE: for some reason, Leaflet requires a height to be set for the map,
+ * but not the width.
+ * It seems like using an SCSS module, you can't target the Leaflet map .leaflet-container
+ * CSS class to override the height.  We may want to have height and width set
+ * by a property on the component.
+ */
+export const MAP_DIV_WIDTH = '800px';
```

Comment:
Are these constants independent of the concrete map  used? If not, please hide them behind the `Coscrad-Leaflet-Map` implementation.